src/torchcodec/decoders/_core/VideoDecoder.cpp

```diff
@@ -234,59 +234,72 @@ VideoDecoder::VideoDecoder(const void* buffer, size_t length) {
 }
 
 void VideoDecoder::initializeDecoder() {
-  // Some formats don't store enough info in the header so we read/decode a few
-  // frames to grab that. This is needed for the filter graph. Note: If this
-  // takes a long time, consider initializing the filter graph after the first
-  // frame decode.
```

Comment:
Curious if we should keep that comment, or what was the reason to remove it? It might be helpful to keep (especially the "Note" part), particularly so that the reader knows that some frame decoding does happen within `avformat_find_stream_info()`?

I'm finding this from the [FFmpeg docs](https://ffmpeg.org/doxygen/6.1/group__lavf__decoding.html)

> Some formats do not have a header or do not store enough information there, so it is recommended that you call the [avformat_find_stream_info()](https://ffmpeg.org/doxygen/6.1/group__lavf__decoding.html#gad42172e27cddafb81096939783b157bb) function which tries to read and decode a few frames to find missing **information.**

Reply:
Good catch. I removed the comment because of the latter part about lazy initialization. But the other stuff is both relevant and informative. I'll submit a PR which revives the relevant stuff.